packages/office-ui-fabric-react/src/components/Check/index.ts

```diff
@@ -1 +1,2 @@
-export * from './Check';
\ No newline at end of file
+export * from './Check';
```

Comment:
export `CheckBase` as well perhaps.

Also did you want to delete the scss file?

Reply:
Alright.

Not yet, there are some rogue components importing it directly. I'll come back to delete once those are converted (namely DetailsList... lol).